graal/backends/core/cocom.py

```diff
@@ -227,6 +255,42 @@ def analyze(self, file_path):
         return lizard_analysis
 
 
+class RepositoryAnalyzer:
+    """Class to analyse the content of a repository"""
+
+    def __init__(self, details=False):
+        self.details = details
+        self.lizard = Lizard()
+
+    def analyze(self, repository_path, files_affected):
+        """Analyze the content of a repository using CLOC and Lizard.
+
+        :param repository_path: repository path
+
+        :returns a list containing the results of the analysis
+        [ {
+          'loc': ..,
```

Comment:
the new attribute `in_commit` is missing in the docstring

Reply:
Yes. Added 👍

Reply:
Thanks @inishchith